README.md

```diff
@@ -87,13 +87,13 @@ After the Maven command completes, the WAR file locates in `${DIR}/cargotracker/
 Create a bash script with environment variables by making a copy of the supplied template:
 
 ```bash
-cp ${DIR}/cargotracker/.scripts/setup-env-variables-template.sh ${DIR}/cargotracker/.scripts/setup-env-variables.sh
+cp ${DIR}/cargotracker/src/test/aks/setup-env-variables-template.sh ${DIR}/cargotracker/src/test/aks/setup-env-variables.sh
 ```
 
-Open `${DIR}/cargotracker/.scripts/setup-env-variables.sh` and enter the following information. Make sure your Oracle SSO user name and password are correct.
+Open `${DIR}/cargotracker/src/test/aks/setup-env-variables.sh` and enter the following information. Make sure your Oracle SSO user name and password are correct.
 
 ```bash
-export WLS_AKS_REPO_REF="278a527cb7d1eb44a5a65ccc02b6837fe42967db" # oracle/weblogic-azure reference
+export WLS_AKS_REPO_REF="2024-03-13-2-Q1" # oracle/weblogic-azure reference
```

Comment:
Update to tag `2024-03-21-1-Q1`.

Reply:
Resolved in https://github.com/Azure-Samples/cargotracker-wls-aks/pull/7/commits/635c0079fb9d5c31d681d07e6cc147bb446393c1